frontend/src/index.jade

```diff
@@ -166,6 +163,9 @@ html
 
     vuetemplate#v_authorship
       #authorship
+        .toolbar(v-if="info.totalCommits > 0")
```

Comment:
This forces `totalCommits` to be present in `info`. However in #524, `totalCommits` has to be derived from repo commits, as it cannot be inside the url.

Reply:
means i take from `window.REPOS` instead of passing it in?

Reply:
Yes, which I have implemented in #524.

Reply:
i saw the implementation, a func to open up `window.REPOS`. the implementation is gud.

but i think i shouldn't pull that change here. because that would require me to pull the function from the javascript file as well, there might be more conflicts.

i put it as it is, but add a comment, marking it to be removed in during merging with #524?